### PR TITLE
Add recipe for gopeaks

### PR DESCRIPTION
**Pull request by gartician**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by gartician** [comment]
Hi conda forge I am willing to be a maintainer of this package.

**Comment by gartician** [comment]
@conda-forge-admin this package is ready for review!

**Comment by gartician** [comment]
@conda-forge-admin, please ping conda-forge/staged-recipes

File: recipes/gopeaks/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/maxsonBraunLab/gopeaks/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 36e13bf083089accbfef3d463bec16922dd5b5ad0e02db408404d6e5b361a80a # cmd: curl -sL {{ url }} | openssl sha256
+
+build:
+  skip: win
```
Comment (carterbox) [inline recipes/gopeaks/meta.yaml:14]:
```suggestion
  skip:  # [win]
```

File: recipes/gopeaks/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  script: mkdir -p {{ PREFIX }}/bin && GOOS=linux GOARCH=amd64 go build -o {{ PREFIX }}/bin/ && chmod a+x {{ PREFIX }}/bin/gopeaks
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('go') }}
+  host:
+    - go
```
Comment (carterbox) [inline recipes/gopeaks/meta.yaml:22]:
```suggestion
  build:
    - {{ compiler('go') }}
    - go-licenses
```

**Review by carterbox — CHANGES_REQUESTED** [review]
I'm pretty sure you aren't including all of the licenses for the dependencies which have been statically linked into this project. Please see this issue for more details:

https://github.com/conda-forge/conda-forge.github.io/issues/1052

**Comment by gartician** [comment]
Hi @carterbox thanks for your help with this PR, I think this package is a better fit for the bioconda channel since this is a bioinformatics program. Thank you for your time, and I will close this PR!

**Pull request closed without merging**